desc/optimize/optimizer.py

```diff
@@ -316,13 +320,22 @@ def jac_wrapped(x_reduced):
             df = objective.jac(x)
             return df[:, unfixed_idx] @ Z
 
+        def is_unnested_wrapped(x_reduced):
```

Comment:
eq is an input here, so we could just use its is_nested method

Reply:
Ah wow that is obvious but I did not notice that, I'll change this and get rid of the is_unnested fxn in objective_funs